src/tmpfiles/tmpfiles.c

```diff
@@ -3669,12 +3671,19 @@ static int parse_line(
                         u = user;
 
                 r = find_uid(u, &i.uid, uid_cache);
-                if (r < 0) {
+                if (r == -ESRCH && arg_graceful) {
+                        _cleanup_free_ char *un = uid_to_name(getuid());
+                        if (!un)
+                                return log_oom();
+
+                        log_syntax(NULL, LOG_DEBUG, fname, line, r,
+                                   "Failed to resolve user '%s', using '%s' instead: %m", u, un);
+                        missing_user_or_group = true;
```

Comment:
so the message is a bit of a lie, because this only uses the user mentioned in the log message if the inode was so far missing. for existing inodes it won#t do a thing though if `uid_set = false` is set...

i.e. rework the message to `…, not adjusting ownership of '%s'"`?

Reply:
Done.